docs/user-guide/cli-usingcli.md

```diff
@@ -31,41 +32,331 @@ zowe --ac
+
+- **Service profiles:** Store connection information for specific mainframe service, such as IBM z/OSMF. Plug-ins can introduce other service profile types, such as the `cics` profile to connect to IBM CICS.
+
+- **Base profiles:** Store connection information for use with one or more services. Your service profiles can pull information from base profiles as needed, so that you can specify a common username and password once. The base profile can optionally store tokens to connect to Zowe API Mediation Layer, which improves security by enabling Multi-Factor Authentication (MFA) and Single Sign-on (SSO).
+
+**Tips:**
+- You can have multiple service profiles and multiple base profiles.
+- Profiles are **not** required. You can choose to specify all connection details for every command.
+- Profile values are stored on your computer in plaintext by default, in the `C:\Users\<yourUsername>\.zowe\profiles` folder.
+
+### Displaying profile help
+
+Use help to learn about options for creating profiles. For example, for a `zosmf` profile, issue the following command:
+
+```
+zowe profiles create zosmf-profile --help
+```
+
+### Service profiles
+
+Create profiles that target a specific mainframe service, then use profiles to issue commands. For example, issue the following command (substituting your connection details) to create a `zosmf` service profile named `myprofile123`:
+
+```
+zowe profiles create zosmf-profile myprofile123 --host host123 --port port123 --user ibmuser --password pass123
+```
+
+Use the profile. For example, issue the following command to list all data sets under the name `ibmuser` on the system that you specified in your profile:
+
+```
+zowe zos-files list data-set "ibmuser.*" --zosmf-profile myprofile123
+```
+
+**Note:** If you do not specify a profile, your default profile is used. The first profile that your create is your default. You can set a profile as your default with the `zowe profiles set-default` command.
+
+### Base profiles
+
+Base profiles store your connection details and provide them to service profiles and commands as needed. The base profile can also contain a token to connect to services through API ML.
+
+For example, if you use the same username and password across multiple services, you can create a base profile with your username and password. After the base profile is created, you can omit the `--username` and `--password` options when you issue commands or service profiles such as `zosmf` and `tso`. Commands will use the values provided by the base profile.
+
+<!--
+TODO Insert example syntax here for creating base profile manually and issuing a command.
```

Comment:
```
zowe profiles create base <profileName> --user <user> --password <pass>
zowe files ls ds <dsname> --host <host>
```
It isn't necessary to supply user and password as CLI options on the 2nd command, because they are present in the base profile.